src/rpc/util.h

```diff
@@ -236,6 +238,7 @@ class RPCHelpMan
     std::string ToString() const;
     /** If the supplied number of args is neither too small nor too high */
     bool IsValidNumArgs(size_t num_args) const;
+    inline void Check(const JSONRPCRequest& request) const;
```

Comment:
in commit 2512d3d2e2f2a2b891f9a3baa1a1309edaf41e01:

Missing doc string and could make `IsValidNumArgs` private?

Reply:
Done!